app/components/user-summary.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import EntryGroup from '../models/entry-group';
+
+const { get } = Ember;
+
+export default Ember.Component.extend({
+  tagName: 'section',
+  classNames: ['header-nav-section right header-user-summary-container'],
```

Comment:
I would have split the class names into an array instead of a single string, but :ok: